### DISCUSSION TITLE
Layers keep deactivating

### DISCUSSION THREAD

**Discussion by laruniBIL** [opengisch/QField#4196] [Q&A]
I have quite the urgent (and very basic, sry) question:

My colleague is in the field right now trying to work with QField. As he is far away I cannot really help him, the problem is as follows:

Whenever he opens QField and the project, all the layers I have loaded him into the project are deactivated. He then has to activate all of them (activating the groups, expanding the groups, upping opacity) and then activate all of the shapes that are included below as well (through long click on the feature and then setting a green tick.

After doing this, the map works fine. But whenever he closes QField and reopens the project, its back to the beginning and he has to restart all of it.

Is there anything I can advice him on how to solve this? I am at the end of my very miniscule knowledge...

Any help is greatly appreciated!

**Comment by nirvn** [comment]
@laruniBIL , what version of QField are they using?

**Reply by laruniBIL** [reply]
Sadly, I've got no idea, sorry! 

I think the problem first arose after changing the map theme, but now we cannot fix it, no matter the map theme we're in

**Reply by nirvn** [reply]
Ok well there's little we can do without more information. Sorry.

**Reply by laruniBIL** [reply]
Sad to hear. Thanks a lot for trying to help, anyway

I'll try to ask him about it, but due to his age he isn't very adept when it comes to the digital realm, thus I am not sure how to lead him there.

Have you ever heard of such a problem before?
